interactive_engine/compiler/src/test/java/com/alibaba/graphscope/cypher/antlr4/WithTest.java

```diff
@@ -184,4 +184,22 @@ public void with_10_test() {
                     + "], matchOpt=[INNER])",
                 project.explain().trim());
     }
+
+    @Test
+    public void with_11_test() {
+        RelNode project =
+                Utils.eval("Match (a:person)-[]-(b:person) Return [a.name, b.age, 1]").build();
```

Comment:
how about returning an empty array like `[ ]`

Reply:
> how about returning an empty array like `[ ]`

We have the same behavior as official Neo4j, both of which output an empty list.
```
@neo4j> Match (a) Return [];
+-----+
| $f0 |
+-----+
| []  |
| []  |
| []  |
| []  |
| []  |
| []  |
+-----+
```